x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -110,6 +110,12 @@ interface GetAlertParams {
   index?: string;
 }
 
+interface GetAlertsCountParams {
+  id?: string;
```

Comment:
Do we use this `id` somewhere?
```
public async getAlertsCount({ featureIds, filter }: GetAlertsCountParams)
```

Reply:
Absolutely not, it is a leftover from a prev implementation, good catch on it!